recipe_scrapers/_schemaorg.py

```diff
@@ -111,12 +111,12 @@ def ingredients(self):
         ]
 
     def nutrients(self):
-        nutrients = self.data.get("nutrition")
-        return [
-            normalize_string(nutrient) + ": " + normalize_string(value)
+        nutrients = self.data.get("nutrition") or {}
```

Comment:
`get()` allows a second parameter of what to return if the key you want doesn't exist. Is this what you want or are you checking if the value of `nutrition` is `Falsey`?